README.md

```diff
@@ -6,13 +6,24 @@ This is my educational project within IT KPI Python mentorship program by @webkn
 
 ## Installation
 
-Soon
+1. Clone this repository:
+    ```console
+    $ git clone git@github.com:kpi-web-guild/django-girls-blog-Martolivna.git
+    ```
 
+2. Install requirements:
+    ```console
+    $ pip install -r requirements/base.in -c requirements/base.txt
+    ```
 
 
 ## Usage
 
-Soon
+Start the server:
+    ```console
+    $ python manage.py runserver
+    ```
```

Comment:
The indentation here causes poor/broken rendering: you can preview it here https://github.com/kpi-web-guild/django-girls-blog-Martolivna/blob/features/16-create-django-project/README.md#usage.

Dedent it and possibly add an empty line before the code block.

Reply:
Fixed fef8570486136c8525d68159ee8e741c5f31baff